### PR TITLE
server connects to DB with IAM auth

### PR DESCRIPTION
**Pull request by andrew-nowak**
Set up the server with the ability and config to connect to the database with IAM auth. When running locally you can connect to the CODE db with an `ssm ssh` tunnel. There's currently no way to connect it to a local postgres, we'll need to figure out some way to switch between local<->local, and local<->CODE, but that's for the future.

Instructions for running with tunnel in the README.

### CODE REVIEW

File: newswires/app/conf/Database.scala [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  //       "databaseName" -> "birthdays",
+  //       "password" -> password
+  //     )
+  //
+  //   Configuration.from(
+  //     Map(
+  //       "slick.dbs.default.db.properties" -> configuration
+  //     )
+  //   )
+  // }
```
Comment (twrichards) [inline newswires/app/conf/Database.scala:47]: is this intentionally commented with the intention of reinstating soon for local postgres access (i.e. displaying what ingestion lambda is producing)

File: newswires/client/src/App.tsx [hunk 2]
```diff
@@ -1,9 +1,229 @@
+import { css } from '@emotion/react';
+import { useCallback, useEffect, useMemo, useState } from 'react';
```
Comment (twrichards) [inline newswires/client/src/App.tsx:2]: might as well remove `useCallback` as unused

File: newswires/client/src/App.tsx [hunk 3]
```diff
@@ -1,9 +1,229 @@
...
+
+	const [query, setQuery] = useState<string>('');
+
+	const [selected, setSelected] = useState<WireData | undefined>(undefined);
+
+	const safeBodyText = useMemo(() => {
+		return selected?.content.body_text
+			? sanitizeHtml(selected.content.body_text)
+			: undefined;
+	}, [selected]);
```
Comment (twrichards) [inline newswires/client/src/App.tsx:62]: we probably ought to add quite a strict Content Security Policy (like we have for https://github.com/guardian/editorial-wires) before this hits production, to guard against malicious content (e.g. XSS) that might find its way into wire content (as it's presented on our domain). we should consider baking in the hash of our own JS bundle such that it's the only JS allowed to run on the page.

**Review by twrichards — APPROVED** [review]
nice work 🙌 
(couple of non-blocking comments)

File: newswires/app/conf/Database.scala (hunk 1 above)
Comment (andrew-nowak) [reply newswires/app/conf/Database.scala:47]: yep exactly

File: newswires/client/src/App.tsx (hunk 3 above)
Comment (andrew-nowak) [reply newswires/client/src/App.tsx:62]: sounds good to me! That is exactly why I pass the content through `sanitize-html` first, but no reason not to do both - let's do this in a followup PR though

**Pull request merged**